test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -39,6 +40,11 @@ const (
 var (
 	stackManager     *StackManager
 	initStackManager sync.Once
+
+	// Sets the log level for Pulumi operations
+	// Be careful setting this value, as it can expose sensitive information in the logs.
+	// https://www.pulumi.com/docs/support/troubleshooting/#verbose-logging
+	pulumiLogLevel = flag.Int("pulumi-log-level", 1, "Pulumi log level")
```

Comment:
I would not use a flag. We have the Param store that gets value for this purpose (taking from env vars)

Reply:
What about allowing using flags as a fallback for env variables in store ?

Reply:
IMHO it does not really make sense, we already have flags in the `invoke` command. I don't think adding flags to `go test` is user friendly.

Reply:
updated to use runner params instead. also added a `logToStdErr` option.